CHANGELOG.md

```diff
@@ -1,10 +1,15 @@
+## [2.1.1]
+### Changed
+* `DatabaseMapping`s no longer make copies of the Thrift objects, it mutates the original objects instead.
+* Upgrade Spring, BeeJU and other dependencies
+
 ## [2.1.0] 2017-10-03
 ### Changed
 * Changed the default GC settings, less heap, more reserved percentage, works better with large requests.
 * Upgrade Hive from 2.1.0 to 2.3.0.
-* Depend on latest parent with test.arguments build parameter
+* Depend on latest parent with test.arguments build parameter.
```

Comment:
Nice, well spotted on the missing full stops! ;) However I have now noticed that you are missing one on line 4!

Reply:
I'll add it